app/src/main/java/com/android/voyageur/ui/profile/EditProfileScreen.kt

```diff
@@ -160,3 +317,14 @@ fun EditProfileScreen(userViewModel: UserViewModel, navigationActions: Navigatio
             }
       })
 }
+
+fun Context.findActivity(): ComponentActivity {
+  var context = this
+  while (context is ContextWrapper) {
+    if (context is ComponentActivity) {
+      return context
+    }
+    context = (context as ContextWrapper).baseContext
```

Comment:
This here is marked for me as useless cast, maybe you could remove it.

Reply:
You are right! Thank you!